lib/linguist/languages.yml

```diff
@@ -1036,6 +1036,7 @@ Clojure:
   - ".cljscm"
   - ".cljx"
   - ".hic"
+  - ".bb"
```

Comment:
You're going to need to move this up to above `.boot` as extensions should be alphabetical after the first extension which is considered the primary extension.